recipes/meld

```diff
@@ -0,0 +1 @@
+(abc-mode :fetcher github :repo "benjaminprogram/meld")
```

Comment:
`abc-mode` -> `meld`

And there is no repository at https://github.com/benjaminprogram/meld . (404 response)